module/Dashboard/src/Dashboard/Model/Dao/NewRelicDao.php

```diff
@@ -569,7 +569,12 @@ public function fetchThreshold(array $params = array())
      */
     private function convertTimeBetweenTimezones($time, $originTimezone, $targetTimezone, $format = 'Y-m-d H:i:s')
     {
-        $convertedTime = new \DateTime($time, new \DateTimeZone($originTimezone));
+        try {
+            $convertedTime = new \DateTime($time, new \DateTimeZone($originTimezone));
+        } catch (\Exception $e) {
+            $date = \DateTime::createFromFormat('m DD,yy hh:MM meridian', $time);
```

Comment:
If this solves the problem with wrong time format, what about moving this 2 lines to try block and in catch block return for instance empty string ?